testing/tests/DevExpress.ui.widgets/gallery.tests.js

```diff
@@ -740,6 +740,13 @@ QUnit.module('behavior', {
 
         assert.equal(instance._pagesCount(), 2, 'pages count is correct');
     });
+
+    QUnit.test('resizeCallback is called after item is rendered (T1132935)', function(assert) {
+        const gallery = this.$element.dxGallery({
+            items: [0, 1, 2, 3],
+        }).dxGallery('instance');
+        assert.equal(gallery._wasAnyItemTemplateRendered, true);
```

Comment:
can we do it without a private api?

Reply:
i tried the Callbacks.fired, but it was not fired
i tried to add callback that sets cache variable to resizeCallbacks and it didnt work too and i havent found the similar cases in test code, so i decided to leave it as it is

Reply:
probably if we use `triggerResizeEvent` instead of direct `_dimensionChanged` call it will be easier to test